packages/@webex/internal-plugin-voicea/src/voicea.ts

```diff
@@ -340,19 +343,25 @@ export class VoiceaChannel extends WebexPlugin implements IVoiceaChannel {
 
   /**
    * request turn on Captions
+   * @param {string} languageCode
    * @returns {Promise}
    */
-  private requestTurnOnCaptions = (): undefined | Promise<void> => {
+  private requestTurnOnCaptions = (languageCode?): undefined | Promise<void> => {
```

Comment:
Shouldn't we just return without doing anything if languageCode is not provided?

Reply:
No. If we do not have the languageCode specified, voicea will take en as default and that is why we have it optional